modules/internal/String.chpl

```diff
@@ -427,6 +427,75 @@ module String {
       return ret;
     }
 
+    /* This function takes an array of stings as input.
+
+    returns:  The longest common prefix path of all the input paths
+    */
```

Comment:
I'd prefer you to use the comment specified in the issue as the documentation for this function (though you will need to update for the array argument)